person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    age > 18
+  end
+
+  def can_use_services?
+    of_age? && parent_permission
```

Comment:
**OPTIONAL** - According to the project requirements this function should `return true if person is of age or if they have permission from parents`